libs/wingc/src/type_check.rs

```diff
@@ -770,6 +788,8 @@ impl Display for Type {
 			Type::MutJson => write!(f, "MutJson"),
 			Type::Nil => write!(f, "nil"),
 			Type::Unresolved => write!(f, "unresolved"),
+			// TODO should probably just be called `uninferred`
```

Comment:
It's used when we are never able to make an inference, then this type is left and is still visible in errors or the LSP.

I removed the comment but set the name as `unknown` because it's short and I didn't like how `uninferred` looked.